src/main/java/com/jagrosh/jmusicbot/commands/dj/RemoveRangeCmd.java

```diff
@@ -0,0 +1,107 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.audio.AudioHandler;
+import com.jagrosh.jmusicbot.audio.QueuedTrack;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import com.jagrosh.jmusicbot.queue.FairQueue;
+
+/**
+ * Command that provides users the ability to remove a range of songs in
+ * the queue.
+ */
+public class RemoveRangeCmd extends DJCommand 
+{
+    /** 
+    * RemoveRangeCmd Class constructor.
+    */
+    public RemoveRangeCmd(final Bot bot)
+    {
+        super(bot);
+        this.name = "removerange";
+        this.help = "removes a range of songs from the queue";
+        this.arguments = "<from> <to>";
+        this.aliases = bot.getConfig().getAliases(this.name);
+        this.beListening = true;
+        this.bePlaying = true;
+    }
+
+    
+    /** 
+     * Perform the command of removing a range of songs from the queue.
+     * 
+     * @param event the event that is being handled
+     */
+    @Override
+    public void doCommand(final CommandEvent event) 
+    {
+        final AudioHandler handler = (AudioHandler)event.getGuild().getAudioManager().getSendingHandler();
+        final FairQueue<QueuedTrack> queue = handler.getQueue();
+        if(queue.isEmpty())
+        {
+            event.replyError("There is nothing in the queue!");
+            return;
+        }
+
+        final String[] parts = event.getArgs().split("\\s+", 2);
+
+        if(parts.length < this.arguments.split("\\s+").length)
+        {
+            event.replyError("Please include two valid indexes.");
+            return;
+        }
+        
+        int fromIndex;
+        int toIndex;
+        
+        try
+        {
+            // Validate the args
+            fromIndex = Integer.parseInt(parts[0]);
+            toIndex = Integer.parseInt(parts[1]);        
+            if (fromIndex > toIndex)
+            {
+            	event.replyError("From position cannot be greater than the to position.");
+            	return;
+            }
+
+            // Validate that from and to are available
+            if (isUnavailablePosition(queue, fromIndex))
+            {
+            	final String reply = String.format("`%d` is not a valid position in the queue!", fromIndex);
+            	event.replyError(reply);

```

Comment:
We generally don't use `String.format` in this project. This statement can also be simplified:
```suggestion
            	event.replyError(fromIndex+" is not a valid position in the queue!");
```